### PR TITLE
Allow for selecting sheet by number

### PR DESCRIPTION
**Pull request by dakira**
This small (non-backwards-breaking) change allows for selecting the actual sheet to be read by the reader.

```php
SimpleExcelReader::create('file.xlsx')
    ->take(5)
    ->fromSheet(2)
    ->getRows();
```

If you enter a sheetNumber that doesn't exist, the closest sheet will be used. An addition could be to throw an exception if the sheet doesn't exist, but I wanted to keep it as simple as possible.

### CODE REVIEW

**Comment by freekmurze** [comment]
Could you throw an exception in that case by default, and add a test to prove that your code works?

**Comment by dakira** [comment]
Will do! :-)

**Comment by dakira** [comment]
@freekmurze There you go, hope this fits the standard. :-)

File: src/SimpleExcelReader.php [hunk 1]
```diff
@@ -268,6 +272,23 @@ protected function getValueFromRow(Row $row): array
...
+        $this->reader->open($this->path);
+
+        foreach ($this->reader->getSheetIterator() as $key => $sheet) {
+            if ($key === $this->sheetNumber) {
+                break;
+            }
+        }
+
+        if ($this->sheetNumber !== $key) {
+            throw new InvalidArgumentException("Sheet {$key} does not exist in {$this->path}.");
```
Comment (freekmurze) [inline src/SimpleExcelReader.php:286]: 👍

**Comment by freekmurze** [comment]
Thank you!

**Pull request merged**